crawler/extractData.py

```diff
@@ -21,12 +21,98 @@ def extractData(baseURL):
   description = metaDesc.get('content') if metaDesc!=None else 'No description provided.'
 
   body=''
-  for node in soup.findAll(['p', 'a']):
+  for node in soup.findAll(['p', 'a', 'div', 'li']):
     body+=node.text+'\n'
 
```

Comment:
Right here, do this:

outlinks = set()
for node in soup.findAll('a'):
  href = link.get('href')
  outlinks.add(href)